res/values-de/strings.xml

```diff
@@ -6,43 +6,49 @@
+    <string name="downloadableDictionaries">Wörterbücher zum Herunterladen</string>
+    <string name="sortDicts">Wörterbücher sortieren</string>
     <string name="downloadButton">%.1fMB</string>
     <string name="updateAvailable">Aktualisierung!</string>
     <string name="moveToTop">Nach Anfang ziehen</string>
     <string name="deleteDictionary">Wörterbuch entfernen</string>
     <string name="indexInfo">%1$s: %2$,d Wörter</string>
     <string name="managerShow">Zeigen</string>
     <string name="managerHide">Verstecken</string>
-    <string name="error">Error</string>
-    <string name="unableToReadDictionaryDir">Unable to read dictionary directory: \n%1$s\n\nAndroid external storage directory: \n%2$s</string>
+    <string name="error">Fehler</string>
+    <string name="unableToReadDictionaryDir">Fehler beim Lesen des Wörterbuch Verzeichnis: \n%1$s\n\nAndroid externer Speicher Verzeichnis: \n%2$s</string>
 
     <!-- Download. -->
-    <string name="downloadFailed">Download failed: \n%s.</string>
-    <string name="unzippingDictionary">Unzipping dictionary: \n%s</string>
-    <string name="unzippingFailed">Unzipping failed: \n%s</string>
-    <string name="installationFinished">Installation finished: \n%s.</string>
-    <string name="notWritable">Dictionary location %s not writable, change in settings.</string>
+    <string name="downloadFailed">Download fehlgeschlagen: \n%s.</string>
+    <string name="downloadManagerQueryFailed">Konnte Download Manager nicht beauftragen.\nIst er deaktiviert oder fehlt er?</string>
+    <string name="unzippingDictionary">Wörterbuch entpacken: \n%s</string>
+    <string name="unzippingFailed">Entpacken fehlgeschlagen: \n%s</string>
+    <string name="installationFinished">Installation beendet: \n%s.</string>
+    <string name="notWritable">Wörterbuch Verzeichnis %s ist nicht beschreibbar. Bitte ändern Sie dies in den Einstellungen.</string>
 
     <!-- DictionaryActivity -->
     <string name="searchText">Suchtext</string>
     <string name="selectDictionary">Wörterbuch wählen…</string>
     <string name="addToWordList">Zum Wortliste hinzufügen: %s</string>
     <string name="searchForSelection">Suche: %s</string>
     <string name="failedAddingToWordList">Fehler beim Hinzufügen zur Wortliste: %s</string>
-    <string name="invalidDictionary">Invalid dictionary: file=%1$s, error=%2$s</string>
+    <string name="invalidDictionary">Fehlerhaftes Wörterbuch: Datei=%1$s, Fehler=%2$s</string>
     <string name="noSearchResults">Kein Treffer.</string>
     <string name="aboutDictionary">Über Wörterbuch…</string>
-    <string name="fontFailure">Font failure: %s</string>
+    <string name="fontFailure">Schriftfehler: %s</string>
     <string name="seeAlso">vgl.: %1$s (%2$s)</string>
     <string name="speak">Sprechen</string>
+    <string name="speak_left">Speak left side</string> //TODO
+    <string name="speak_right">Speak right side</string> //TODO
```

Comment:
@mueller-ma Hat sich jetzt eh erledigt, aber der Grund für weglassen ist, daß Tools wie Android lint und verschiedene Übersetzungstools wie zum Beispiel poedit das einfach und zuverlässig verarbeiten, wohingegen solche pseudo-Übersetzungen und TODOs eher Probleme machen.
Ist zwar weniger kritisch weil keines dieser Tools bisher gut genug funktionert, daß ich es wirklich verwenden würde, aber das ist der Hauptgrund...
Ein anderer ist, daß syntaktisch korrekte Kommentare in XML etwas unschön sind (// macht genaugenommen das XML ungültig).